products/opensuse/main.pm

```diff
@@ -411,6 +411,7 @@ else {
         || is_systemd_test())
     {
         loadtest "console/system_prepare";
+        loadtest "x11/disable_screensaver" if (check_var('FLAVOR', 'NET') && check_var('UPGRADE', '1') && heck_var('ORIGINAL_VERSION', '15.0'));
```

Comment:
Should be `check_var` instead of `heck_var`:
`&& heck_var('ORIGINAL_VERSION'`

Reply:
Thanks, fixed